### ISSUE TITLE
Acess Web UI directly with Tesla Browser

### ISSUE THREAD

**Issue by JakobLichterfeld** [TheRealKasumi/TesLight#54]
As a user I do want to access the WebUI directly with the Tesla Browser.

**Comment by JakobLichterfeld** [comment]
Tesla Browser blocks direct IP calls, so we must maskerade it via NAT or force special IPs like 3.3.3.3 for the WI-FI spot.

**Comment by TheRealKasumi** [comment]
Unfortunately it doesn't seem to be so easy. Assigning an IP address to the AP that is not blocked by the browser isn't enough. The car is rejecting the whole WiFi connection. I also think that there is a check if the car can access the internet or Tesla's servers. Otherwise conenction is rejected again. At least that is my observation so far.

**Comment by JakobLichterfeld** [comment]
Yeah, was just for getting it out of my head. First brainstorming.
There are ways to make it happen, I will digg deeper.

**Comment by TheRealKasumi** [comment]
Yes I think it should definitely be possible. I would actually think it would be quite easy with an Raspi or laptop/pc. Not sure how much we are limited on the ESP32. I think here it could get a little tricky. But I like the idea. Unfortunately no time so far to really look into it 😋 .
